journal-letters/modernized/03-chapter-1.adoc

```diff
@@ -129,15 +129,15 @@ occasioned wrath,) and would sprinkle our hearts from an evil conscience,
 and wash us with pure water; that we might draw near with a true heart,
 in full assurance of faith, as the Christians of old did. Heb. 10:22.
 
-But, alas! we had not boldness,--for the living faith was lacking;
+But, alas--we had not boldness,--for the living faith was lacking;
```

Comment:
I think we want to leave the exclamation point in, and _then_ add the double dash.  Like: `But, alas!--we had`